### DISCUSSION TITLE
Validate entered coordinates based on geography

### DISCUSSION THREAD

**Discussion by carlosmbe** [specify/specify7#5837] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Talk to Theresa for more context.

They would like it if Specify checked if the Lat/Long Fields entered match the area they want.

For example, if I put 35.9078° N, 127.7669° E and call it Tokyo, Japan. I'd want to check that's easy and built into Specify that confirms that it is Tokyo Japan. For the record, those coordinates are wrong and are actually for South Korea.

**Describe the solution you'd like**
I put the notes in the screen shots.

# 1
<img width="1204" alt="Screenshot 2023-07-03 at 4 06 44 PM" src="https://github.com/specify/specify7/assets/53784701/109410b1-5952-4452-8d0b-26b987ff0987">

# 2
<img width="1202" alt="Screenshot 2023-07-03 at 3 53 12 PM" src="https://github.com/specify/specify7/assets/53784701/019b5fcd-7e9d-4d92-97b6-cb3ad79ad209">

**Reported By**
Jerusalem 

---

Edited by @grantfitzsimmons:
---

## We should not ever automatically fill coordinates based on Geography.

_# 3
<img width="100" alt="Screenshot 2023-07-03 at 4 00 39 PM" src="https://github.com/specify/specify7/assets/53784701/e1c5d875-9dbd-44fd-8c51-e81be19d2182">

<img width="100" alt="Screenshot 2023-07-03 at 4 01 55 PM" src="https://github.com/specify/specify7/assets/53784701/b74a8a25-598a-4d1a-bd04-91acb3112a08">_

**Comment by maxpatiiuk** [comment]
Duplicate of https://github.com/specify/specify7/issues/1940 (but that one is for workbench)
@grantfitzsimmons do you want to merge these issues?

**Comment by grantfitzsimmons** [comment]
> ## Show warning in the WorkBench if coordinates do not match localities #1940
>This would be really helpful when uploading data to the WorkBench to validate that the imported coordinates are correct. This should not block uploading, but rather give a warning of some kind.
>
>*Locality name, as determined by GeoMap or GeoLocate

Copied from https://github.com/specify/specify7/issues/1940
